src/list-seqs.py

```diff
@@ -1,17 +1,27 @@
-import argparse
+import sys
 
+def list_seqs():
 
-def main():
-    argparser = argparse.ArgumentParser(
-        description="Extract the sequences from a simple-fastq file")
-    argparser.add_argument(
-        "fastq",
-        type=argparse.FileType('r')
-    )
-    args = argparser.parse_args()
+    inFile = sys.argv[1]  
+    with open(inFile,'r') as j:
+        lines = j.readlines()
 
-    print(f"Now I need to process the records in {args.fastq}")
+    record_list = []
+    header = ''
+    sequence = ''
+    for line in lines:
+        line = line.strip()
+        if line.startswith('@'):
+            if header != "":
+                record_list.append([header, sequence])
+                sequence = ""
+            header = line[1:]
+        else:
+            sequence = sequence + line
+    record_list.append([header, sequence])   
 
-
-if __name__ == '__main__':
-    main()
+    for i in record_list:
+        print(i[1]) 
+    
+    return ''
```

Comment:
What is this doing? Why do you `return` outside of a function? That doesn't seem like something a sane person would do...